### PR TITLE
SRX-5VUIF9 Release workflow on tag creation

### PR DESCRIPTION
**Pull request by mnishamk-freiheit**
When a tag is created, automatically create a release with the helm chart.

### CODE REVIEW

File: .github/workflows/execution-plan-tag.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Release
+        uses: ./.github/actions/softprops-action-gh-release
+        with:
+          files: |
+            charts/kuberpult/kuberpult-${{ needs.generate_version.outputs.version }}.tgz
+          body: |
+            Release ${{ needs.generate_version.outputs.version }}
+          tag_name: ${{ needs.generate_version.outputs.version }}
+          generate_release_notes: true
+          # Will be removed once tested
```
[the remark this answers is not in the document]
Comment (mnishamk-freiheit) [reply .github/workflows/execution-plan-tag.yml]: updated

File: charts/kuberpult/Makefile [hunk 2]
```diff
@@ -62,6 +62,9 @@ release: test $(TGZ_FILE)
 	@echo "Please upload $(TGZ_FILE) to the github release"
 	gsutil cp $(TGZ_FILE) gs://fdc-charts
 	
+release-tag: test	$(TGZ_FILE)
```
Comment (sven-urbanski-freiheit-com) [inline charts/kuberpult/Makefile:65]:
```suggestion
release-tag: test $(TGZ_FILE)
```

File: charts/kuberpult/Makefile [hunk 3]
```diff
@@ -62,6 +62,9 @@ release: test $(TGZ_FILE)
 	@echo "Please upload $(TGZ_FILE) to the github release"
```
Comment (sven-urbanski-freiheit-com) [inline charts/kuberpult/Makefile:62]: is the `release` target still necessary?
I think we can remove it.

File: charts/kuberpult/Makefile (hunk 2 above)
Comment (mnishamk-freiheit) [reply charts/kuberpult/Makefile]: updated

File: .github/workflows/execution-plan-tag.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the version
+        id: get_version
+        run: |
+          echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+    outputs:
+      version: ${{ steps.get_version.outputs.VERSION }}
+
+  execution-plan:
```
[the remark this answers is not in the document]
Comment (mnishamk-freiheit) [reply .github/workflows/execution-plan-tag.yml:17]: thats my bad, had removed it to test the pipeline.
Its removed now

File: .github/workflows/execution-plan-tag.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+name: "Release kuberpult"
```
Comment (sven-urbanski-freiheit-com) [inline .github/workflows/execution-plan-tag.yml:1]: Please add a bit of description, e.g.
```suggestion
# This workflow creates a new release in github, when a tag is pushed.
name: "Release kuberpult"
```

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

File: charts/kuberpult/Makefile (hunk 3 above)
Comment (mnishamk-freiheit) [reply charts/kuberpult/Makefile:62]: This is just in case it would be used manually, it still has the correct location to push to

**Pull request merged**